src/py_obs/osc.py

```diff
@@ -49,14 +53,41 @@ async def api_request(
         except aiohttp.ClientResponseError as cre_exc:
             raise ObsException(**cre_exc.__dict__) from cre_exc
 
+    async def get_auth_header(self) -> hash:
```

Comment:
`hash` is not a valid type.
```suggestion
    async def get_auth_header(self) -> dict[str, str]:
```

Reply:
Some puppetized syntax history...